server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -117,6 +117,9 @@ public enum MetadataVersion {
     // Enables share groups. Note, share groups are for preview only in 4.1. (KIP-932).
     IBP_4_1_IV1(27, "4.1", "IV1", false),
 
+    // Enables "streams" groups. Note, streams groups are for early access only in 4.1. (KIP-1071).
```

Comment:
We don't need two MV versions if we aren't adding any new metadata records for this feature. If you don't want to enable it by default, we can remove this one and just keep ibp 4.1-2

Reply:
(Seems like the share groups one was also done incorrectly.)

Reply:
That one is different because it requires an update to the MV log. (See the "true" for the first MV)

Reply:
@AndrewJSchofield : Added a clarification in https://github.com/apache/kafka/pull/19293/files#r2064970001 regarding IBP_4_1_IV1. We could follow up there.